web-graphql-example/README.adoc

```diff
@@ -0,0 +1,14 @@
+= Vert.x Web GraphQL examples
+
+Here you will find examples demonstrating Vert.x Web GraphQL in action.
+
+https://vertx.io/docs/vertx-web-graphql/java/[Vert.x Web GraphQL] extends Vert.x Web with the https://www.graphql-java.com/[GraphQL-Java] library so that you can build a GraphQL server.
+
+== Simple
+
+This example uses the Vert.x Web client to send a GraphQL request and log the response.
+
```

Comment:
You should explain what the example does in more concrete terms, so we understand there are link / user types, etc.

Reply:
Will do